proteomic_aggregation_and_visualization/python/proteomic_aggregation_and_visualization.ipynb

```diff
@@ -0,0 +1,4004 @@
+{
```

Comment:
<p>"The slot&nbsp;<code>in_manifest</code>&nbsp;links NMDC data objects that can be analyzed together."  This is not universally true, can you elaborate to be more specific for proteomics?  I think it would be helpful to show a data object, where you got that ID, and then pull all the other data objects with that ID.  Otherwise you're pulling this id out of thin air with no explanation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>